src/jni/java/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>com.vesoft</groupId>
     <artifactId>nebula-utils</artifactId>
-    <version>1.0.0-rc2</version>
+    <version>1.0.0-rc3</version>
```

Comment:
Have we uploaded rc3 version jni? rc2 is ok I guess

Reply:
I understand that RC3 will release later so I update this version number.
Actually, when RC2 release we should update this version.

Reply:
Ok, remember to upload it to mvn